src/gui/editors/AutomationEditor.cpp

```diff
@@ -398,6 +398,58 @@ void AutomationEditor::drawLine( int x0In, float y0, int x1In, float y1 )
 
 
 
+bool AutomationEditor::fineTuneValue(timeMap::iterator node, bool editingInValue)
+{
+	if (node != m_pattern->getTimeMap().end())
```

Comment:
Style suggestion:

We could save indention by using the inverse here:
```suggestion
	if (node == m_pattern->getTimeMap().end()) { return false; }
```

Then the rest of the block is the normal function indented one less.